README.md

```
@@ -66,6 +66,20 @@ Safety Net will not run on production sites. It will check the `WP_ENVIRONMENT_T
 add_filter( 'safety_net_show_production_notice', '__return_false' );
 ```
 
+## Adding plugins to the Deny list.
+You can add a plugin to the deny list for a single site using the following filter.
+```php
+add_filter( 'safety_net_denylist_plugins', function( $denylist ) {
+    // Add the full path to the plugin file here.
+	$denylist[] = 'plugin-folder/plugin-file.php'; 
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)